### ISSUE TITLE
Release new version with updated dependencies

### ISSUE THREAD

**Issue by ryandesign** [stefansl/image-shrinker#66]
Image Shrinker's package.json says:

https://github.com/stefansl/image-shrinker/blob/8bca16946ce1861d81ccc89a9fe36294ddf3dca3/package.json#L65

My understanding of package.json syntax is that this will use any svgo version up to but not including 2.0.0.

Image Shrinker 1.6.4 was released October 2020 at which time the current version of svgo was 1.3.2. Today in June 2023 the current version of svgo is 3.0.2.

Maybe it's time to release a new version of Image Shrinker with updated versions of svgo and any other of its dependencies that have had new releases?

**Comment by NickDimou** [comment]
This great software needs an update.

**Comment by AlexLaforge** [comment]
I absolutely agree, plus a Mac Arm version

**Comment by ydoupo** [comment]
yes, please update with an Arm Mac Version, this software is sooooo good!

**Comment by stefansl** [comment]
Yes. I need to rewrite the whole code based on Vue 3 and all the new packages besides. That takes a lot of time. But it's on my list 😊
